packages/dropdowns/package.json

```diff
@@ -23,6 +23,8 @@
     "@zendeskgarden/container-selection": "^0.2.3",
     "classnames": "^2.2.5",
     "downshift": "^3.2.7",
+    "moment": "^2.24.0",
```

Comment:
I worry about the date picker and therefore moment being included in vendor bundles even if it is not referenced. 

Moment is huge https://bundlephobia.com/result?p=moment@2.24.0 especially with locale support on top. I would much prefer a new package that build on top of react-dropdowns that was focused around handling calendar/date/time.

Reply:
The original plan to include this with dropdowns was to reduce the amount of CSS bloat for the dropdown related styling. `react-popper` and the menu positioning/animation styling would be duplicated.

Is that the correct bundle size? That looks like the size of Moment.js with _all_ locales included. Minified it should be around 16KB which is still hefty.

Reply:
You might be right, about the size - it would be great if you check it.

I think it would be great if getting a styled react-popper would just be something this package would offer to other packages, as that seems to be something that would happen multiple times. That was exactly what the Popover was in the old components.

I just think that it is a mistake to mix these things, as I see the amount of calendar related functionality grow a lot.

Reply:
<img width="789" alt="Screenshot 2019-06-27 at 23 06 42" src="https://user-images.githubusercontent.com/90802/60300877-4795f280-9930-11e9-9fa3-1a18c40c6bcb.png">

You need this as far as I can tell:

```
plugins: [
  new webpack.IgnorePlugin(/^\.\/locale$/, /moment$/)
]
```

https://github.com/moment/moment/issues/2416

Reply:
Resolving with the latest push.